frontend/coprs_frontend/coprs/views/backend_ns/backend_general.py

```diff
@@ -286,16 +286,34 @@ def build_task_canceled(task_id):
 @backend_ns.route("/pending-actions/")
 def pending_actions():
     'get the list of actions backand should take care of'
+    waiting_actions = actions_logic.ActionsLogic.get_waiting()
+    busy_namespaces = set()
+    for action in waiting_actions:
+        if (
+            action.object_type == ActionObjectTypeEnum.COPR
+            and action.action_type == ActionTypeEnum("delete")
+        ):
+            busy_namespaces.add(action.copr.full_name)
```

Comment:
Just a note that `action.copr._` will be accessible once #2805 is merged

Reply:
Can you rebase this PR on top of the branch in #2805?